client/data/marketplace/constants.ts

```diff
@@ -0,0 +1,10 @@
+export const BASE_STALE_TIME = 1000 * 60 * 60 * 2; // 2 hours
+export const DEFAULT_PAGE_SIZE = 24;
+export const DEFAULT_FIRST_PAGE = 1;
+export const DEFAULT_CATEGORY = 'all';
+
+export const SITE_SEARCH_ENDPOINT =
+	'https://public-api.wordpress.com/rest/v1/sites/108986944/search';
```

Comment:
Any chance you add a comment or a different constant explaining this value `108986944`?

Reply:
Good idea to make it more clear, I've added it as a constant! 🙂